README.rst

```diff
@@ -109,14 +114,14 @@ and overrides the ``configure()`` method to define all of the application compon
       createDataset(CUBE_NAME, Cube.class, props);
 
       addFlow(new CubeWriterFlow());
-      addService(SERVICE_NAME, new CubeHandler());
+      addService(new CubeService());
```

Comment:
If you add a method to the API so that the Name and Description could be set at the same time, then you could go with just the handler. I agree, handler would be better.
See https://issues.cask.co/browse/CDAP-2287